org-formation/600-access/_tasks.yaml

```diff
@@ -74,6 +74,18 @@ JCWorkflowsNextflowDevAdmin:
     Account: !Ref WorkflowsNextflowDevAccount
     Region: us-east-1
 
+WorkflowsNextflowProdServiceAccount:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/master/templates/IAM/service-account.yaml
+  StackName: workflows-nextflow-prod-service-account
```

Comment:
maybe leave off `prod` if you want this same service account deployed to both dev and prod accounts?